niworkflows/utils/spaces.py

```diff
@@ -410,7 +406,7 @@ def check_space(space):
                 space = (None, )
 
             space = space[0]
-        return Space(space, spec)
+        return Reference(space, spec)
 
     def __init__(self, spaces=None):
```

Comment:
while updating the fmriprep docs, i realized there isn't a way to initiate a cached `SpatialReference`.  WDYT about adding an additional keyword `checkpoint` to call `self.checkpoint()` if spaces were set?

Reply:
Oh, yes. I also had thought of this. Will do!